hw/ip/otbn/otbn.core

```diff
@@ -30,6 +35,7 @@ targets:
   default: &default_target
     filesets:
       - files_rtl
+      - files_model
```

Comment:
Can you gate that fileset with `tool_verilator ? (files_model)`? This should make regressions with VCS (as in private ci) pass.